jobs/user_activity_card_gold.py

```diff
@@ -0,0 +1,69 @@
+"""
```

Comment:
Please change the script name to rohan_cards_gold.py. Each person will have their own table and needs to be uniquely identifiable.

Reply:
@sravantit25 I am done with all suggested changes. I re-based my branch with master so you will notice multiple file changes, please look at latest commit for your simplicity.

Reply:
@rohandudam Thank you Rohan. Looks good. Merged the PR.